units/Makefile

```diff
@@ -15,32 +15,27 @@ clean:
 # Clean and rebuild everything
 rebuild: clean all
 
-# Extract preferred_unit annotations from schema (fast: ~1-2 seconds)
-output/schema_preferred_units.tsv: ../nmdc_schema/nmdc_materialized_patterns.yaml
-	@mkdir -p output
-	poetry run units-schema-extract --schema-file $< --output $@
-
-# Convert preferred units to UCUM notation (fast: ~1-2 seconds)
-output/schema_ucum_input.tsv output/schema_ucum_detailed.tsv: output/schema_preferred_units.tsv
-	poetry run units-schema-convert --input $< --output output/schema_ucum_input.tsv --detailed output/schema_ucum_detailed.tsv
-
-# Generate yq commands for storage_units annotations (fast: ~1-2 seconds)
-output/yq_commands_single_unit.txt output/yq_commands_multi_unit.txt: output/schema_ucum_input.tsv
-	poetry run units-schema-generate --input $< --output-dir output
+# Removed misleading preferred_unit analysis targets (schema_extract, schema_convert, schema_generate)
+# These reported "problem units" that were already addressed by storage_units annotations
+# See: https://github.com/microbiomedata/nmdc-schema/issues/2687
 
 
 # Production data validation (uses MongoDB YAML dump)
-# ⚠️  SLOWEST TARGET: Processes 32,000+ QuantityValue instances from GB-sized MongoDB dump
+# ⚠️ UNIMPLEMENTED: Script units-production-validate does not exist
+# TODO: Create units/scripts/production_validate.py and add entry point to pyproject.toml
+# Expected implementation: Processes 32,000+ QuantityValue instances from GB-sized MongoDB dump
 # Expected time: Minutes to hours depending on dump size
+#
 # Set SCHEMA_FILE to control which schema version to validate against:
 # ENV=dev (default): ../nmdc_schema/nmdc_materialized_patterns.yaml (current development schema)
 # ENV=prod: ../local/nmdc_schema_last_release.yaml (latest release schema)
 SCHEMA_FILE ?= $(if $(filter prod,$(ENV)),../local/nmdc_schema_last_release.yaml,../nmdc_schema/nmdc_materialized_patterns.yaml)
 
-slow-outputs/production_validation_results.tsv: ../local/mongo_via_api_as_unvalidated_nmdc_database.yaml
-	@mkdir -p slow-outputs
-	poetry run units-production-validate --input $< --output $@ --schema-file $(SCHEMA_FILE)
-	@echo "Note: Requires MongoDB dump from: make local/mongo_via_api_as_unvalidated_nmdc_database.yaml"
+# DISABLED - missing script units-production-validate
+# slow-outputs/production_validation_results.tsv: ../local/mongo_via_api_as_unvalidated_nmdc_database.yaml
+# 	@mkdir -p slow-outputs
+# 	poetry run units-production-validate --input $< --output $@ --schema-file $(SCHEMA_FILE)
+# 	@echo "Note: Requires MongoDB dump from: make local/mongo_via_api_as_unvalidated_nmdc_database.yaml"
```

Comment:
I'm tempted to say just remove all this. 

I can see the value in being able to make a change to the schema locally and then see if that change would make any existing records in MongoDB invalid. That general use case makes sense to me. But:

1. I think it should just be a generic process based on standard LinkML validation (plus our custom plugin) and not anything specific to units.
2. I think we should encourage (and make it easy) having this process talk to a local MongoDB database (presumably containing data from a recent production backup) instead of pounding our production API to produce huge YAML files that are going to be slower to validate anyway.

Anyway, all that is better captured in an issue than in comments here.